### PR TITLE
repoquery: accept package names specified as NEVRA

### PR DESCRIPTION
**Pull request by spacekpe**

### CODE REVIEW

**Comment by rholy** [comment]
We agreed that writing tests against plugins might be more work that writing the plugins itself. However, it isn't impossible. So if you don't have time to write them, fix at least the other comments, please.

**Comment by rholy** [comment]
We agreed that writing tests against plugins might be more work that writing the plugins itself. However, it isn't impossible. So if you don't have time to write them, fix at least the other comments, please.

**Comment by jsilhan** [comment]
@spacekpe, now it solves BZ, append to commit message `(RhBug:1179366)` and replace in https://github.com/rpm-software-management/dnf-plugins-core/blob/master/doc/repoquery.rst `pkg-name` with `package-spec` and reference http://dnf.readthedocs.org/en/latest/command_ref.html#specifying-packages , please.

**Comment by spacekpe** [comment]
I did doc changes but I really don't have time for tests, sorry.

File: doc/repoquery.rst [hunk 1]
```diff
@@ -38,7 +38,11 @@ Description
 Select Options
 --------------
 
-Together with ``<pkg-name>``, control what packages are displayed in the output. If ``<pkg-name>`` is given, the set of resulting packages is limited to the ones with a matching name (globbing supported), else all packages are considered.
+Together with ``<pkg-spec>``, control what packages are displayed in the output. If ``<pkg-spec>`` is given, the set of resulting packages is limited to the ones with a matching name (globbing supported), else all packages are considered.
```
Comment (rholy) [inline doc/repoquery.rst]: Not just "a matching name"...

**Comment by rholy** [comment]
Pylint raises:
```
C0330: repoquery:192,0: Wrong hanging indentation.
                    self.base.sack, with_provides=False)
                |   ^
```
Please, fix the indentation.

**Comment by spacekpe** [comment]
Docs and pylint nit fixed.

**Comment by spacekpe** [comment]
We found out that base.sack object in CliStub is missing so I'm currently not able to write tests for the new functionality, sorry!

**Comment by mluscon** [comment]
No issues spotted during functionality testing.

**Comment by jsilhan** [comment]
Merged, thanks

**Pull request closed without merging**